packages/node_modules/@ciscospark/react-component-activity-item-base/src/index.js

```diff
@@ -92,7 +92,7 @@ function ActivityItemBase({
       <div className={classNames(getActionClassNames({highlight: isFlagged, flagActionPending: isFlagPending}))}>
         <Icon
           ariaLabel="Flag for follow-up"
-          color="black"
+          color="black-100"
```

Comment:
what does 100 means?

Reply:
This was due to a warning message that was displayed:

> getColorValue.js:71 [@momentum-ui] Design system colors update: black is now black-100. Refer to https://momentum.design/styles/color/style

Reply:
After discussing with the Momentum team, this warning is a bug and it should still be `black`. Will revert.